src/managers/StageInstanceManager.js

```diff
@@ -0,0 +1,150 @@
+   */
+
+  /**
+   * Creates a new stage instance.
+   * @param {CreateStageInstanceOptions} options The options to create the stage instance
+   * @returns {Promise<StageInstance>}
+   * @example
+   * // Create a stage instance
+   * guild.stageInstances.create({
+   *  channel: '1234567890123456789',
+   *  topic: 'A very creative topic',
+   *  privacyLevel: 'GUILD_ONLY'
+   * })
+   *  .then(stageInstance => console.log(stageInstance))
+   *  .catch(console.error);
+   */
+  async create(options) {
+    if (typeof options !== 'object') throw new TypeError('INVALID_TYPE', 'options', 'object', true);
+    let { channel, topic, privacyLevel } = options;
+    const channelID = this.guild.channels.resolveID(channel);
+    if (!channelID) throw new Error('STAGE_CHANNEL_RESOLVE');
+
+    if (privacyLevel) privacyLevel = typeof privacyLevel === 'number' ? privacyLevel : PrivacyLevels[privacyLevel];
+
+    const data = await this.client.api['stage-instances'].post({
+      data: {
+        channel_id: channelID,
+        topic,
+        privacy_level: privacyLevel,
+      },
+    });
+
+    return this.add(data);
+  }
+
+  /**
+   * Fetches the stage instance associated with a stage channel, if it exists.
+   * @param {StageChannel|Snowflake} channel The stage channel whose instance is to be fetched
+   * @param {BaseFetchOptions} [options] Additional options for this fetch
+   * @returns {Promise<StageInstance>}
+   * @example
+   * // Fetch a stage instance
+   * guild.stageInstances.fetch('1234567890123456789')
+   *  .then(stageInstance => console.log(stageInstance))
+   *  .catch(console.error);
+   */
+  async fetch(channel, { cache = true, force = false } = {}) {
+    const channelID = this.guild.channels.resolveID(channel);
+    if (!channelID) throw new Error('STAGE_CHANNEL_RESOLVE');
+
+    if (!force) {
+      const existing = this.cache.find(stageInstance => stageInstance.channelID === channelID);
+      if (existing) return existing;
+    }
+
+    const data = await this.client.api('stage-instances', channelID).get();
+    return this.add(data, cache);
+  }
+
+  /**
+   * Options used to update an existing stage instance.
+   * @typedef {Object} UpdateStageInstanceOptions
+   * @property {string} [topic] The new topic of the stage instance
+   * @property {PrivacyLevel|number} [privacyLevel] The new privacy level of the stage instance
+   */
+
+  /**
+   * Updates an existing stage instance.
+   * @param {StageChannel|Snowflake} channel The stage channel whose instance is to be updated
+   * @param {UpdateStageInstanceOptions} options The options to update the stage instance
+   * @returns {Promise<StageInstance>}
+   * @example
+   * // Update a stage instance
+   * guild.stageInstances.update('1234567890123456789', { topic: 'new topic' })
+   *  .then(stageInstance => console.log(stageInstance))
+   *  .catch(console.error);
+   */
+  async update(channel, options) {
```

Comment:
Wouldn't the right method name be `edit`? The only thing that uses `update` for editing are [`PermissionOverwrites`](https://discord.js.org/#/docs/main/master/class/PermissionOverwrites?scrollTo=update), [`MessageComponentInteraction`](https://discord.js.org/#/docs/main/master/class/MessageComponentInteraction?scrollTo=update), and [`ButtonInteraction`](https://discord.js.org/#/docs/main/master/class/ButtonInteraction?scrollTo=update), although the semantics of the last 2 are quite different.

Reply:
yeah, I was going for the `edit` too but the docs call it as `Update Stage Instance` instead of the regular `Modify <Something>`

Reply:
I [asked](https://canary.discord.com/channels/613425648685547541/697489244649816084/853896503721984020) in Discord Developers regarding this, and turns out it was an oversight, I opened a PR in the docs fixing that: https://github.com/discord/discord-api-docs/pull/3122